docker-compose.yml

```diff
@@ -120,18 +120,47 @@ services:
     extra_hosts:
       - "host.docker.internal:host-gateway"
 
+  # Different FerretDB v1 instances must use different POSTGRES_DB to avoid error such as
+  # `relation "_ferretdb_database_metadata" already exists (SQLSTATE 42P07)`.
```

Comment:
Seems like we need multiple databases for v1 by setting `POSTGRES_DB=dance`. For postgres-documentdb we must use `POSTGRES_DB=postgres` for documentdb extension to be initialized.

It seems simplifying to one postgresql isn't possible